react/features/base/icons/svg/lead-record.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="40" height="40" viewBox="0 0 40 40" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
we should probably remove the `fill` values for these svgs, since we usually expect it to be provided from outside